src/server/handlers/consents/{ID}/revoke.ts

```diff
@@ -45,6 +45,10 @@ import { Enum } from '@mojaloop/central-services-shared'
 import { consentDB } from '../../../../lib/db'
 import { Consent } from '../../../../model/consent'
 
+/**
+ * Asynchronously deals with validating request, revoking consent object
+ * and send outgoing PATCH consent/{id}/revoke request to switch
+ */
 export async function validateRequestAndRevokeConsent (
   request: Request): Promise<void> {
   const consentId = request.params.id
```

Comment:
in L77 remove `if` guard which should be implemented in `revokeConsentStatus` and properly catch thrown exception using PUT ../error